### PR TITLE
Release v1.0.0

### PR DESCRIPTION
**Pull request by ryanchristo**

### CODE REVIEW

File: .env.local.example [hunk 1]
```diff
@@ -1,2 +1,4 @@
 VITE_LOCAL_HOSTNAME=http://127.0.0.1
-VITE_PROXY_URL=http://localhost:5000
+VITE_PROXY_URL_REGEN_MAINNET=http://api.registry.regen.network
+VITE_PROXY_URL_REGEN_TESTNET=http://api-staging.registry.regen.network
+VITE_INDEXER_GRAPHQL_API=http://localhost:5000/indexer/graphql
```
Comment (wgwz) [inline .env.local.example:4]:
```suggestion
VITE_INDEXER_GRAPHQL_API=http://localhost:5000/indexer/v1/graphql
```

File: indexer-codegen.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { CodegenConfig } from '@graphql-codegen/cli'
+
+const config: CodegenConfig = {
+  overwrite: true,
+  schema: 'http://localhost:5000/indexer/graphql',
```
Comment (wgwz) [inline indexer-codegen.ts:5]:
```suggestion
  schema: 'http://localhost:5000/indexer/v1/graphql',
```

File: netlify.toml [hunk 3]
```diff
@@ -1,3 +1,9 @@
+[context.production.environment]
+  VITE_INDEXER_GRAPHQL_API="https://api.registry.regen.network/indexer/graphql"
```
Comment (wgwz) [inline netlify.toml:2]:
```suggestion
  VITE_INDEXER_GRAPHQL_API="https://api.regen.network/indexer/v1/graphql"
```

File: netlify.toml [hunk 4]
```diff
@@ -1,3 +1,9 @@
...
+
+[context.deploy-preview.environment]
+  VITE_INDEXER_GRAPHQL_API="https://api-staging.registry.regen.network/indexer/graphql"
```
Comment (wgwz) [inline netlify.toml:5]:
```suggestion
  VITE_INDEXER_GRAPHQL_API="https://api-staging.regen.network/indexer/v1/graphql"
```

**Review by wgwz — APPROVED** [review]
utACK

File: .env.local.example (hunk 1 above)
Comment (ryanchristo) [reply .env.local.example:4]: Opened a pr so we can add these to the dev branch as well: https://github.com/regen-network/groups-ui/pull/138

File: indexer-codegen.ts (hunk 2 above)
Comment (ryanchristo) [reply indexer-codegen.ts:5]: https://github.com/regen-network/groups-ui/pull/138

File: netlify.toml (hunk 3 above)
Comment (ryanchristo) [reply netlify.toml:2]: https://github.com/regen-network/groups-ui/pull/138

File: netlify.toml (hunk 4 above)
Comment (ryanchristo) [reply netlify.toml:5]: https://github.com/regen-network/groups-ui/pull/138

**Pull request merged**